ui/src/pages/ResetPassword.tsx

```diff
@@ -105,13 +74,13 @@ export function ResetPasswordPage(): JSXElement {
                   icon={<i class="fa-solid fa-key" />}
                 />
               )}
-            </ResetPassword.Field>
+            </Field>
```

Comment:
Do you think maybe eventually we could also abstract away the TextInput stuff etc for forms? Based on input type?

Reply:
Sort of as a generic `Form` component?

Reply:
Maybe